server/util.go

```diff
@@ -157,3 +160,31 @@ func checkBootstrapRequest(clusterID uint64, req *pdpb.BootstrapRequest) error {
 
 	return nil
 }
+
+func postJSON(client *http.Client, url string, data []byte, checkOpts ...func([]byte, int)) error {
```

Comment:
`checkOpts` is used for testing purposes. Besides, I think we can avoid making an internal HTTP call.

Reply:
I don't want making an internal HTTP call either, but api/handler do this before. My original intention is to use the existing handler logic and avoid reimplementing the logic of add scheduler. But `addEvictOrGrant` post a request I don't know why such an internal post was done before

Reply:
what's your suggestion?

Reply:
@rleungx and I had a brief voice exchange: 1. Move `postJSON` to `apiutil` to avoid redundant code caused by circular dependencies but continue to use the existing interface 2. Reimplement add scheduler logic independent of handler logic.
Because the latter is too complicated and needs to maintain two sets of similar logic, for now use the existing handler interface. cc @JmPotato